### PR TITLE
Update authentication.md

### PR DESCRIPTION
**Pull request by jong-int**
Updating getting started doc

### CODE REVIEW

File: docs/graphql-concepts/fragments.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+Here's an example of defining a fragment and using it in a query: 
+
+```
+query 
+  queryName {
+    fieldName1
+    fieldName2 {
+      ...fragmentName
+    }
+  }
```
Comment (bcole) [inline docs/graphql-concepts/fragments.md:29]:
```suggestion
query queryName {
  fieldName1
  fieldName2 {
    ...fragmentName
  }
}
```
Formatting fix, this is how we usually format GraphQL Queries

File: docs/graphql-concepts/fragments.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+Now define the fragment and corresponding set of fields: 
+
+```
+fragment 
+  fragmentName on FieldName {
+    field1
+    field2
+    field3
+  }
```
Comment (bcole) [inline docs/graphql-concepts/fragments.md:41]:
```suggestion
fragment fragmentName on FieldName {
  field1
  field2
  field3
}
```

File: docs/graphql-concepts/pagination.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+        hasNextPage
+        startCursor
+        endCursor
+        hasPreviousPage
+      }
+      field1
+      field2
+    }
+  }
+```ß
```
Comment (bcole) [inline docs/graphql-concepts/pagination.md:35]:
````suggestion
```
````

File: docs/graphql-concepts/pagination.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+     pageInfo {
+        hasNextPage
+        startCursor
+        endCursor
+        hasPreviousPage
+      }
+      field1
+      field2
+    }
+  }
```
Comment (bcole) [inline docs/graphql-concepts/pagination.md:34]:
```suggestion
query queryName {
  fieldName1
  fieldName2(pagination: {first: numOfRecords, after: cursorValue}) {
   pageInfo {
      hasNextPage
      startCursor
      endCursor
      hasPreviousPage
    }
    field1
    field2
  }
}
```

File: docs/graphql-concepts/pagination.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+```ß
+
+In this case, the server will only count records following the `cursorValue`. **Tip**: You can [use introspection](../../graphql-concepts/introspection/) to see which fields support pagination.
```
Comment (bcole) [inline docs/graphql-concepts/pagination.md:37]:
```suggestion
In this case, the server will only count records following the `cursorValue`. **Tip**: You can [use introspection](./introspection/) to see which fields support pagination.
```

File: docs/graphql-concepts/query.md [hunk 6]
```diff
@@ -5,25 +5,29 @@ nav_order: 1
...
 
-Feel free to play around with different queries.  Many GraphQL IDEs have an autocomplete feature, to make forming queries easier!
+```
+query 
+  getCompanyName {
+    company {
+     companyName
+     companyType
+    }
+  }
```
Comment (bcole) [inline docs/graphql-concepts/query.md:29]:
```suggestion
query getCompanyName {
  company {
    companyName
    companyType
  }
}
```

File: docs/graphql-concepts/query.md [hunk 7]
```diff
@@ -5,25 +5,29 @@ nav_order: 1
...
+```
+The server will return values for the companyName and companyType fields.
```
Comment (bcole) [inline docs/graphql-concepts/query.md:31]:
```suggestion
The server will return values for the `companyName` and `companyType` fields. 
```

File: docs/graphql-concepts/variables.md [hunk 8]
```diff
@@ -0,0 +1,42 @@
...
+
+```
+query 
+  queryName ($variable : type) {
+    fieldName1
+    fieldName2 (filter: {field1 {in : $variable}}) {
+      field1
+      field2
+   }
+  }
```
Comment (bcole) [inline docs/graphql-concepts/variables.md:29]:
```suggestion
query queryName ($variable : type) {
  fieldName1
  fieldName2 (filter: {field1 {in : $variable}}) {
    field1
    field2
  }
}
```

File: docs/getting-started/getting-started.md [hunk 9]
```diff
@@ -1,11 +1,11 @@
 ---
 layout: default
-title: Getting Started
+title: Get Started
```
Comment (bcole) [inline docs/getting-started/getting-started.md:3]: I don't think I have a strong preference one way or another (`Getting Started` vs `Get Started`).

If we want to make the change to `Get Started` - can we fully make the change?

1. Fix child pages to have parent of `Get Started` (currently all child pages are broken in the navbar)
2. Change URL permalink to `/docs/get-started/`
3. Change any link references throughout to `/get-started/...`

I can help make these changes if you want, but just first wanted to confirm we want to refactor this to "Get Started"

File: docs/getting-started/authentication.md [hunk 10]
```diff
@@ -1,30 +1,66 @@
 ---
 layout: default
-title: Authentication
+title: Set up and authenticate your app
```
Comment (bcole) [inline docs/getting-started/authentication.md:3]: Preference - can we keep sidebar titles short?  It seems long, 2 lines on the sidebar
Screen Shot 2021-04-20 at 11 30 08 AM

The page content can have the longer title if we want, but trying to keep the sidebar links short if possible

File: docs/getting-started/authentication.md [hunk 11]
```diff
@@ -1,30 +1,66 @@
...
 nav_order: 1
 parent: Getting Started
 ---
 
-## Authentication
+# Set up and authenticate your app
 
-To make API calls, you'll need:
+Let's get started. Intuit Ecosystem API gives you incredible flexibility to build creative solutions around key business problems. You can create client or server apps from the ground up, or [start with common use cases](https://intuitdeveloper.github.io/intuit-api-docs/docs/use-cases/).
```
Comment (bcole) [inline docs/getting-started/authentication.md:10]:
```suggestion
Let's get started. Intuit Ecosystem API gives you incredible flexibility to build creative solutions around key business problems. You can create client or server apps from the ground up, or [start with common use cases](../../use-cases/). 
```

File: docs/getting-started/authentication.md [hunk 12]
```diff
@@ -1,30 +1,66 @@
...
 
-### 1. QuickBooks company
+## Step 1: Get to know the QuickBooks API platform
+Already familiar with Intuit Ecosystem API and GraphQL? Feel free to skip this section. 
 
-Depending on your use case (will you be making Payroll API calls?), you'll need a QuickBooks company setup appropriately (with Payroll, for example).  Partners should reach out to their Intuit point-of-contact to get a company set up.
+If you're new, check out these resources so you can develop with confidence:
 
-### 2. An Intuit application
+* [Learn about Intuit Ecosystem API](https://intuitdeveloper.github.io/intuit-api-docs/docs/faq/graphql-vs-rest/)
```
Comment (bcole) [inline docs/getting-started/authentication.md:19]:
```suggestion
* [Learn about Intuit Ecosystem API](../../faq/graphql-vs-rest/)
```

Comment (bcole) [reply docs/getting-started/authentication.md:19]: Is this the right link title - "Learn about Intuit Ecosystem API"?  The link itself takes you specifically to a page about details of GraphQL vs REST, not really generic information about "Intuit Ecosystem API"

I'm actually not sure if linking to that page is helpful at this point, learning about GraphQL vs REST seems like something specific if people are interested later on, not as an intro to our Ecosystem API

File: docs/getting-started/authentication.md [hunk 13]
```diff
@@ -1,30 +1,66 @@
...
+* [Learn about GraphQL](https://intuitdeveloper.github.io/intuit-api-docs/docs/graphql-concepts/)
```
Comment (bcole) [inline docs/getting-started/authentication.md:20]:
```suggestion
* [Learn about GraphQL](../../graphql-concepts/)
```

File: docs/getting-started/authentication.md [hunk 14]
```diff
@@ -1,30 +1,66 @@
...
+4. If you're starting with a sandbox environment, go to the **Development** section and select **Keys & OAuth**. If you're authenticating a production app, go to the **Production** section and select **Keys & OAuth**.
+5. Copy the **Client ID** and **Client Secret**. 
+
+Use the Client ID and Client Secret to [set up OAuth2.0](https://developer.intuit.com/app/developer/qbo/docs/develop/authentication-and-authorization/oauth-2.0). This process grants access tokens so your app can make requests and get responses from our servers.
+
+> **Tip**: Want to try the authentication process with demo code? Check out the [OAuth 2.0 playground](https://developer.intuit.com/app/developer/qbo/docs/develop/authentication-and-authorization/oauth-2.0-playground).
+
+## Step 6: Test GraphQL queries
+
+Use a [GraphQL Integrated Developer Environment (IDE)](https://intuitdeveloper.github.io/intuit-api-docs/docs/getting-started/graphql-ide/) to test queries during development. You can use Insomnia or GraphiQL.
```
Comment (bcole) [inline docs/getting-started/authentication.md:60]:
```suggestion
We suggest using a [GraphQL Integrated Developer Environment (IDE)](../graphql-ide/), like Insomnia or GraphiQL, to test queries during development.
```
They don't have to use Insomnia or GraphiQL, those are just two options.  In theory they could use CURL and not even use an IDE (wouldn't recommend it).  Just proposing a slight rephrase to make it clear that these are options, not mandates.

File: docs/getting-started/authentication.md [hunk 15]
```diff
@@ -1,30 +1,66 @@
...
+
+## Step 7: Develop your app
+
+Now your app is onboarded and authenticated. You have everything you need to build and test queries!
+
+Check out [GraphQL best practices](https://intuitdeveloper.github.io/intuit-api-docs/docs/faq/best-practices/) to develop fast, robust, and reliable apps.
```
Comment (bcole) [inline docs/getting-started/authentication.md:66]:
```suggestion
Check out [GraphQL best practices](../../faq/best-practices/) to develop fast, robust, and reliable apps.
```

File: docs/graphql-explorer/graphql-explorer.md [hunk 16]
```diff
@@ -0,0 +1,11 @@
+---
+layout: default
+title: GraphQL Explorer
+nav_order: 1
```
[the remark this answers is not in the document]
Comment (jong-int) [reply docs/graphql-explorer/graphql-explorer.md:4]: Yes sorry, i meant it to be under home. Let me update this.

File: docs/graphql-explorer/graphql-explorer.md [hunk 17]
```diff
@@ -0,0 +1,11 @@
...
+has_children: true
+permalink: /docs/graphql-explorer/
+---
+
+# GraphQL Explorer
```
[the remark this answers is not in the document]
Comment (jong-int) [reply docs/graphql-explorer/graphql-explorer.md:9]: Great, let's use use that title, that makes sense to me!  

I agree with a description too. Currently I didn't put anything in, I was just trying to create the space for them, then add the wording once it all looks good on the site.

File: docs/use-cases/use-cases.md [hunk 18]
```diff
@@ -6,6 +6,8 @@ has_children: true
 permalink: /docs/use-cases/
 ---
 
-# Use Cases
+# Build apps around use-cases
```
[the remark this answers is not in the document]
Comment (jong-int) [reply docs/use-cases/use-cases.md:9]: nope that's a typo

File: docs/use-cases/use-cases.md [hunk 19]
```diff
@@ -6,6 +6,8 @@ has_children: true
...
 
-The following pages give some sample queries that can be helpful for common use cases.
\ No newline at end of file
+If you already know what you want to develop, here's [how to create queries](../../graohql-explorer/) to fetch data or perform operations around a bespoke experience.
```
[the remark this answers is not in the document]
Comment (jong-int) [reply docs/use-cases/use-cases.md:11]: typo sorry

File: docs/use-cases/use-cases.md [hunk 20]
```diff
@@ -6,6 +6,8 @@ has_children: true
...
 ---
 
-# Use Cases
+# Build apps around use-cases
 
-The following pages give some sample queries that can be helpful for common use cases.
\ No newline at end of file
+If you already know what you want to develop, here's [how to create queries](../../graohql-explorer/) to fetch data or perform operations around a bespoke experience. 
+
+On the other hand, if you need examples to see what's possible, we've got you covered. Here are common use cases for common small business accounting problems.
```
[the remark this answers is not in the document]
Comment (jong-int) [reply docs/use-cases/use-cases.md:13]: Great point, let's just take out accounting. That let's it stay generic.

File: docs/graphql-concepts/graphql-concepts.md [hunk 21]
```diff
@@ -1,25 +1,31 @@
 ---
 layout: default
-title: GraphQL Concepts
+title: Learn GraphQL Concepts
```
[the remark this answers is not in the document]
Comment (jong-int) [reply docs/graphql-concepts/graphql-concepts.md:3]: I am fine with keeping it just GraphQL Concepts. It's something I need more quantitative data to determine. Some readers prefer single words, some prefer actionable titles. I am actually leaning back towards your direction. 

Something to think about as we integrate, nothing I would sweat for the beta site.

File: docs/graphql-concepts/graphql-concepts.md [hunk 22]
```diff
@@ -1,25 +1,31 @@
...
-- __Hierarchical__: A GraphQL query is structured just like the data it returns. It naturally follows relationships between fields and helps to present this data in a user interface.
-- __Language-independent__: GraphQL can be used with any backend framework or programming language implementation.
+## What are some of GraphQL's advantages?
+
+- **Get only the data you need**: GraphQL queries are highly customizable. You specify the fields and values you want the server to return. Apps only get the data it queries for - there’s no over-fetching or under-fetching.
+- **Develop with any language**: GraphQL is language-independent. You can use it with any backend framework or programming language implementation.
+- **Validate queries as you work**: GraphQL servers define the type system. That means everything about it is part of the schema. You can validate queries as you create them with [testing tools](https://intuitdeveloper.github.io/intuit-api-docs/docs/getting-started/graphql-ide/) or [introspection](https://intuitdeveloper.github.io/intuit-api-docs/docs/graphql-concepts/introspection/). You can also let the server validate requests against the current schema.
```
[the remark this answers is not in the document]
Comment (jong-int) [reply docs/graphql-concepts/graphql-concepts.md:25]: Sorry about this, I tried to follow the (../) convention on my own wasn't consistent. Definitely want to follow best practices.

File: docs/graphql-concepts/graphql-concepts.md [hunk 23]
```diff
@@ -1,25 +1,31 @@
...
+- **Always have the current schema**: GraphQL is self-documenting. You can [make an introspection query](https://intuitdeveloper.github.io/intuit-api-docs/docs/graphql-concepts/introspection/) any time you want to see all possible fields and operations for the latest version. 
+- **Create simple data hierarchies**: GraphQL requests and responses are structured the same way and naturally follow relationships between attributes.
+
+Check out [GraphQL.org](https://graphql.org/){:target="_blank"} for a deepr dive.
+
+Also check out: [courses from edx.org](https://www.edx.org/course/exploring-graphql-a-query-language-for-apis?index=product&queryID=dce5c7752e45f9b4ffb33a5af4d0c25e&position=1){:target="_blank"}.
```
[the remark this answers is not in the document]
Comment (jong-int) [reply docs/graphql-concepts/graphql-concepts.md:31]: Yes, i will update

File: docs/graphql-concepts/fragments.md (hunk 1 above)
Comment (jong-int) [reply docs/graphql-concepts/fragments.md:29]: thank you for letting me know, I will not change it in the future, my apologies

File: docs/getting-started/getting-started.md (hunk 9 above)
Comment (jong-int) [reply docs/getting-started/getting-started.md:3]: No I won't mess with this

File: docs/getting-started/authentication.md (hunk 12 above)
Comment (jong-int) [reply docs/getting-started/authentication.md:19]: True, we can remove - I am in the process of trying to create a section of that doc that speaks specifically to each framework and then compares. Thought it was better to include it and let people ignore.

File: docs/getting-started/authentication.md (hunk 14 above)
Comment (jong-int) [reply docs/getting-started/authentication.md:60]: Thank you, this is great feedback.

**Review by bcole — APPROVED** [review]
Looks good, approving - thanks for making the changes!

**Pull request merged**